RHIDP-2140: updates to rbac documentation (#264)
redhat-developer/red-hat-developers-documentation-rhdh: modules/admin/ref-rbac-permission-policies.adoc

Permission policies in {product} are a set of rules to govern access to resources or functionalities. These policies state the authorization level that is granted to users based on their roles. The permission policies are implemented to maintain security and confidentiality within a given environment.

> You can define two types of permissions in {product-short} including, resource type and basic permissions. 
> 
> There are two types of permissions available in {product-short} including, resource type and basic permissions. The distinction between the two permission types depends on whether a permission includes a defined resource type.

Reviewer: Minimalism

```suggestion
You can define the following types of permissions in {product-short}:

* resource type
* basic 

The distinction between the two permission types depends on whether a permission includes a defined resource type.
```

> You can define the resource type permission using either the associated resource type of the permission name. The following is an example of resource type permission definition:

Reviewer: ```suggestion
You can define the resource type permission using either the associated resource type or the permission name as shown in the following example:
```

.Example resource type permission definition
[source,csv]
----
p, role:default/myrole, catalog.entity.read, read, allow
g, user:default/myuser, role:default/myrole

p, role:default/another-role, catalog-entity, read, allow
g, user:default/another-user, role:default/another-role
----

> The basic permission in {product-short} are only defined using the associated name. The following is an example of basic permission definition:

Reviewer: ```suggestion
You can define the basic permission in {product-short} using the permission name as shown in the following example:
```

> 

Reviewer: .Example basic permission definition

[source,csv]
----
p, role:default/myrole, catalog.entity.create, create, allow
g, user:default/myuser, role:default/myrole

> 

Reviewer: ```suggestion
```

Permission policies in {product} are a set of rules to govern access to resources or functionalities. These policies state the authorization level that is granted to users based on their roles. The permission policies are implemented to maintain security and confidentiality within a given environment.

You can define the following types of permissions in {product-short}:

* resource type
* basic

The distinction between the two permission types depend on whether a permission includes a defined resource type.

You can define the resource type permission using either the associated resource type or the permission name as shown in the following example:

> .Example resource type permission definition

Reviewer: We need to give a space in between to make this example label appear in the next line. 

![image](https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/assets/94683525/506ce9c4-d46f-41cb-a1ae-2b0386f0190a)

[source,csv]
----
p, role:default/myrole, catalog.entity.read, read, allow
g, user:default/myuser, role:default/myrole

p, role:default/another-role, catalog-entity, read, allow
g, user:default/another-user, role:default/another-role
----

You can define the basic permission in {product-short} using the permission name as shown in the following example:

> .Example basic permission definition

Reviewer: We need to give a space in between to make this example label appear in the next line.
